README.md

```diff
@@ -1,10 +1,50 @@
 # Log Weasel
 
-Instrument Rails and supported 3rd-party libraries (e.g. Resque, Pwwka) with shared transaction IDs to trace execution of a unit of work across
-applications and application instances.
+Instrument Rails and supported 3rd-party libraries (e.g. Resque, Pwwka) with shared transaction IDs to trace execution of a unit of work across applications and application instances.
```

Comment:
I re-organized this readme somewhat, and normalized whether we line break at 100 char or not (to not).

Reply:
README looks good! Thanks for reorganizing it.